.ci/jobs/pr.yml

```diff
@@ -1,27 +1,27 @@
 ---
 - job:
-    description: Job that runs the pr Jenkinsfile on any PR of the Elastic Cloud on Kubernetes project. This Job is managed by JJB.
+    description: Run ECK tests (unit, integration and E2E smoke test), on every PR
     name: cloud-on-k8s-pr
     project-type: pipeline
     concurrent: true
     pipeline-scm:
       scm:
         - git:
-            url: https://github.com/elastic/cloud-on-k8s
+            url: https://github.com/thbkrkr/cloud-on-k8s
             branches:
               - ${sha1}
             credentials-id: 'f6c7695a-671e-4f4f-a331-acdce44ff9ba'
             refspec: '+refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*'
-      script-path: build/ci/pr/Jenkinsfile
+      script-path: .ci/pipelines/pr.Jenkinsfile
       lightweight-checkout: false
     triggers:
       - github-pull-request:
           org-list:
-            - elastic
+            - thbkrkr
```

Comment:
I guess this should be changed before we merge

Reply:
Absolutely. It comes from "Temporary for testing" (d7be775). It was a desperate attempt to try to test it in real.

Reply:
Reverted.